pom.xml

```diff
@@ -136,13 +136,11 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <maven.compiler.release>11</maven.compiler.release>
-    <maven.compiler.source>11</maven.compiler.source>
-    <maven.compiler.target>11</maven.compiler.target>
```

Comment:
using `maven.compiler.source` and `maven.compiler.target` properties is kind of standard practice. it has the effect to overwrite the Maven compiler plugin properties. I used to know some occasions where this is safer than setting the values in the plugin configuration with a custom property (e.g. java.version).

I think it has to do something with using `-Dmaven.compiler.target=14` which is commonly known by Maven users I guess.

Reply:
I can accept that. will change that to:
```
    <java.version>11</java.version>
    <maven.compiler.release>${java.version}</maven.compiler.release>
    <maven.compiler.source>${java.version}</maven.compiler.source>
    <maven.compiler.target>${java.version}</maven.compiler.target>
```